htdocs/compta/sociales/class/chargesociales.class.php

```diff
@@ -317,7 +317,7 @@ public function update($user, $notrigger = 0)
         $sql.= ", date_ech='".$this->db->idate($this->date_ech)."'";
         $sql.= ", periode='".$this->db->idate($this->periode)."'";
         $sql.= ", amount='".price2num($this->amount, 'MT')."'";
-		$sql.= ", fk_projet='".$this->db->escape($this->fk_project)."'";
+		if($this->fk_project > 0) $sql.= ", fk_projet='".$this->db->escape($this->fk_project)."'";
```

Comment:
If fk_project is empty we must set field to null.

Reply:
Thanks